<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="find_dir_button">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>443</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>560</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restart_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>560</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>restart</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>901</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="MplWidget" name="MplWidget4" native="true">
       <widget class="QPushButton" name="generate4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="MplWidget" name="MplWidget3" native="true">
       <widget class="QPushButton" name="generate3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="MplWidget" name="MplWidget1" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <widget class="QPushButton" name="generate1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MplWidget" name="MplWidget2" native="true">
       <widget class="QPushButton" name="generate2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="run_option">
    <property name="geometry">
     <rect>
      <x>649</x>
      <y>480</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>run</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>511</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="directory">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
